coord/src/dist_aggregate.c

```diff
@@ -89,6 +88,7 @@ static int netCursorCallback(MRIteratorCallbackCtx *ctx, MRReply *rep) {
 
   // Check if an error returned from the shard
   if (MRReply_Type(rep) == MR_REPLY_ERROR) {
+    RedisModule_Log(NULL, "warning", "Coordinator got an error from a shard");
```

Comment:
This was added mainly with the thought of the more interesting errors. It's right that this may add some trivial logs but I think the benefit of having this in the crashes beats that.
We can go for `notice`, since it's the default mode for redis-server.